packages/flame_forge2d/lib/body_component.dart

```diff
@@ -14,18 +14,17 @@ import 'sprite_body_component.dart';
 /// it is a good idea to turn on [debugMode] for it so that the bodies can be
 /// seen
 abstract class BodyComponent<T extends Forge2DGame> extends Component
-    with HasGameRef<T> {
+    with HasGameRef<T>, HasPaint {
```

Comment:
Excellent! I was going to suggest that but I forgot to raise the issue, glad you did it here!